includes/class-wcs-cached-data-manager.php

```diff
@@ -73,16 +73,14 @@ public function cache_and_get( $key, $callback, $params = array(), $expires = WE
 	public function purge_delete( $post_id, $post = null ) {
 		wcs_deprecated_function( __METHOD__, '2.3.0' );
 
-		$post_type = get_post_type( $post_id );
-
-		if ( 'shop_order' === $post_type ) {
+		if ( 'shop_order' === WC_Data_Store::load( 'order' )->get_order_type( $post_id ) ) {
```

Comment:
Won't take an extra mile to rename the `$post_id` variable to `$id` here as this function is deprecated. This change is so that we cannot find `get_post_type()` in our code base.